pom.xml

```diff
@@ -60,15 +60,15 @@
     </repository>
     <repository>
       <id>mvn-gae-plugin-repo</id>
-      <url>http://maven-gae-plugin.googlecode.com/svn/repository/</url>
+      <url>https://maven-gae-plugin.googlecode.com/svn/repository/</url>
     </repository>
     <repository>
       <id>maven-central-rep-1</id>
       <url>https://repo1.maven.org/maven2/</url>
     </repository>
     <repository>
       <id>sun-maven-repository</id>
-      <url>http://download.java.net/maven/2/</url>
+      <url>https://download.java.net/maven/2/</url>
```

Comment:
I don't think either of these repositories still exist. Should just remove them?